src/twig/_twig-components/functions/bem.function.php

```diff
@@ -4,73 +4,44 @@
  * Add "bem" function for Pattern Lab & Drupal
  */
 
-use Drupal\Core\Template\Attribute;
-
-$function = new Twig_SimpleFunction('bem', function ($context, $base_class, $modifiers = array(), $blockname = '', $extra = array()) {
-  $classes = [];
-
-  // If using a blockname to override default class.
-  if ($blockname) {
-    // Set blockname class.
-    $classes[] = $blockname . '__' . $base_class;
-
-    // Set blockname--modifier classes for each modifier.
-    if (isset($modifiers) && is_array($modifiers)) {
-      foreach ($modifiers as $modifier) {
-        $classes[] = $blockname . '__' . $base_class . '--' . $modifier;
-      };
-    }
-  }
-  // If not overriding base class.
-  else {
-    // Set base class.
-    $classes[] = $base_class;
-    // Set base--modifier class for each modifier.
-    if (isset($modifiers) && is_array($modifiers)) {
-      foreach ($modifiers as $modifier) {
-        $classes[] = $base_class . '--' . $modifier;
-      };
-    }
-  }
-
-  // If extra non-BEM classes are added.
-  if (isset($extra) && is_array($extra)) {
-    foreach ($extra as $extra_class) {
-      $classes[] = $extra_class;
-    };
-  }
-
-  if (class_exists('Drupal')) {
-    $attributes = new Attribute();
-
-    // Iterate the attributes available in context.
-    foreach($context['attributes'] as $key => $value) {
-      // If there are classes, add them to the classes array.
-      if ($key === 'class') {
-        foreach ($value as $class) {
-          $classes[] = $class;
-        }
+function bem(\Twig_Environment &$env, $config) {
```

Comment:
Occurs to me that this looks to be re-generated from patternlab and I should find a way to make sure this used going forward

Reply:
This entire function file, or just certain lines of the file?